README.md

```diff
@@ -1,47 +1,85 @@
-more_chests
-===========
+<style>
+    th { display: none; }
+    td { text-align: center; }
+</style>
 
-More Chests
+# more_chests
+Originally a fork of 0gb.us's chests_0gb_us (https://forum.minetest.net/viewtopic.php?f=11&t=4366).  
+Megaf's more_chests fixes several bugs, uses new textures and adds compatibility with [VanessaE's Pipeworks] (https://github.com/VanessaE/pipeworks) mod.  
+nxet's rework introduces a new backend which allows for easier extension of the available models, including two new models showcasing the feature.
 
-This mod is a fork of 0gb.us's chests_0gb_us https://forum.minetest.net/viewtopic.php?f=11&t=4366
 
-Megaf's more_chests fixes several bugs, uses new textures and adds compatibility with [VanessaE's Pipeworks] (https://github.com/VanessaE/pipeworks) mod.
+### Available models
+> NOTE: descriptions of the original models come from 0gb.us initial effort
 
-The following text was writen by 0gb.us
-```
-Cobble Chest:
-{'default:wood','default:cobble','default:wood'},
-{'default:cobble','default:steel_ingot','default:cobble'},
-{'default:wood','default:cobble','default:wood'}
+#### Cobble Chest
+This locked chest looks like cobblestone, and has no info text. Great for hiding things in. However, unlike real cobblestone, this chest is breakable by hand. If you suspect there is one hiding, hold the left mouse button and run your hand along the walls. When cracks appear, you've found the chest.
+1 | 2 | 3
+---|---|---
+default:wood | default:cobble | default:wood
+default:cobble | default:steel_ingot | default:cobble
+default:wood | default:cobble | default:wood
```

Comment:
```suggestion
<table>
    <tbody align="center">
        <tr><td>group:wood</td><td>default:cobble</td><td>group:wood</td>
        <tr><td>default:cobble</td><td>default:steel_ingot</td><td>default:cobble</td>
        <tr><td>group:wood</td><td>default:cobble</td><td>group:wood</td>
    </tbody>
</table>
```

Reply:
This table is valid markdown, though some parsers might prefer padded contents like this:

```
1            | 2              | 3
-------------|----------------|-------------
default:wood | default:cobble | default:wood
```
But I really don't know. The current version is rendered correctly:
https://github.com/minetest-mods/more_chests/blob/52cabc9fbc695a7eb86238e21ec6b6e0ae431c5b/README.md

Reply:
Yes, it is valid, but there is `<style> th { display: none; } td { text-align: center; } </style>` at the top of the rendered page, and it is not supposed to be rendered this way (it is supposed to hide the header, and to center the content of the table instead, and this is not the case).

What I see with the current version:
![image](https://user-images.githubusercontent.com/55180044/94338744-eb1ef880-fff4-11ea-8ca3-1285bcf50d8c.png)

![image](https://user-images.githubusercontent.com/55180044/94338721-c32f9500-fff4-11ea-928f-ea260f5a60f0.png)

Reply:
Just my personal opinion as I stumbled upon this PR: README.md files should not contain HTML or CSS and files should be readable with simple editors that do not render HTML tables for example like `vim` or similar cli editors.
Formatting suggested by @SmallJoker imo is also very good for all use cases.